plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -251,6 +252,34 @@ class HealthChecksFormSection extends Component {
     });
   }
 
+  getIpProtocol(data) {
+    const { healthCheck, key, errors } = data;
+    const runtime = findNestedPropertyInObject(
```

Comment:
Could do but I rather don't because of readability. the function plus parameters are quite long. it is better to grasp if the condition is like `(runtime !== "DOCKER")` instead of `(findNestedPropertyInObject("container.docker.image", data) !=== "DOCKER")`